classes/client/swift_client.php

```diff
@@ -308,6 +308,7 @@ public function define_swift_check($mform, $config) {
 
         } else {
             $mform->addElement('html', $OUTPUT->notification($connection->message, 'notifyproblem'));
+            $permissions = true;
```

Comment:
it this line intended?

Reply:
Following our discussion this will be fixed when issue #155 is solved.